app/controllers/teams_controller.rb

```diff
@@ -63,7 +63,7 @@ def destroy
       format.json { head :no_content }
     end
   end
-  
+
```

Comment:
So I use atom, and these changes are atom automatically removing trailing whitespace.